pkg/allocator/allocator.go

```diff
@@ -23,7 +24,7 @@ import (
 )
 
 var (
-	log = logging.DefaultLogger.WithField(logfields.LogSubsys, "allocator")
+	subsysLogAttr = []any{logfields.LogSubsys, "allocator"}
 )
```

Comment:
Given that this is only used once, it could be directly used inline, rather than having a separate variable.